src/generators/web/build/css.mjs

```diff
@@ -0,0 +1,62 @@
+import { readFile } from 'node:fs/promises';
+
+import { bundleAsync } from 'lightningcss';
+
+/**
+ * Rolldown plugin to compile `.module.css` files using lightningcss
+ * and emit a single bundled CSS file.
+ *
+ * TODO(avivkeller): Once Rolldown supports CSS Modules natively, we
+ * can remove this
+ *
+ * @returns {import('rolldown').Plugin}
+ */
+export default () => {
+  const cssChunks = new Set();
+
+  return {
+    name: 'css-loader',
+
+    /**
+     * Handles loading and transforming CSS module files
+     * @param {string} id - The file path being loaded
+     */
+    async load(id) {
+      if (!id.endsWith('.module.css')) {
+        return null;
+      }
+
+      const source = await readFile(id, 'utf8');
+
+      const { code, exports } = await bundleAsync({
+        filename: id,
+        code: Buffer.from(source),
+        cssModules: true,
+      });
+
+      cssChunks.add(code.toString());
+
+      const mappedExports = Object.fromEntries(
+        Object.entries(exports).map(([k, v]) => [k, v.name])
+      );
+
+      return {
+        code: `export default ${JSON.stringify(mappedExports)};`,
+        moduleType: 'js',
+      };
+    },
```

Comment:
```suggestion
    load: {
      filter: { include: /\.module\.css$/ },
      async handler(id) {
        const source = await readFile(id, 'utf8');

        const { code, exports } = await bundleAsync({
          filename: id,
          code: Buffer.from(source),
          cssModules: true,
        });

        cssChunks.add(code.toString());

        const mappedExports = Object.fromEntries(
          Object.entries(exports).map(([k, v]) => [k, v.name])
        );

        return {
          code: `export default ${JSON.stringify(mappedExports)};`,
          moduleType: 'js',
        };
      },
```

Reply:
Thanks for the suggestion! I'll test the code above, and push it ASAP

Reply:
@TheAlexLichter It appears the `filter. include`-ing isn't properly excluding non-CSS files. _All_ files seem to be passed to the handler, which, in turn, fails to process, since they aren't CSS.

Reply:
Ahh I see, it needs to be `filter.id.include`.